src/entropies_estimators/order_statistics/AlizadehArghami.jl

```diff
@@ -48,14 +48,15 @@ the [`Vasicek`](@ref) estimate ``\\hat{H}_{V}(\\bar{X}, m, n)``, plus a correcti
 See also: [`entropy`](@ref), [`Correa`](@ref), [`Ebrahimi`](@ref),
 [`Vasicek`](@ref), [`DifferentialEntropyEstimator`](@ref).
 """
-@Base.kwdef struct AlizadehArghami{I<:Integer} <: DiffEntropyEst
+@Base.kwdef struct AlizadehArghami{I<:Integer, B} <: DiffEntropyEst
     m::I = 1
+    base::B = 2
 end
 
-function entropy(e::Shannon, est::AlizadehArghami, x::AbstractVector{T}) where T
+function entropy(est::AlizadehArghami, x::AbstractVector{<:Real})
     (; m) = est
     n = length(x)
     m < floor(Int, n / 2) || throw(ArgumentError("Need m < length(x)/2."))
-    h = entropy(Shannon(base = ℯ), Vasicek(; m), x) + (2 / n)*(m * log(2))
-    return h / log(e.base, ℯ)
+    h = entropy(Vasicek(; m, base = est.base), x) + (2 / n)*(m * log(2))
```

Comment:
Be careful here! The base needs to be `ℯ` for the `Vasicek` estimator, because the correction term uses the natural log (and I'm not sure if it would be valid for any other base without modification). The `est.base` call should only happen in the final normalization to target unit, on line 61.

Reply:
oh that's why the test of this estimator were failing by such a large margin lol

Reply:
Given that the above comment is fixed, and if tests pass and docs look good, nothing should be broken by doing this switch `definition.base`->`est.base`. But make sure the tests actually do pass, and that the plots look good.